.gitignore

```diff
@@ -44,3 +44,6 @@ datadog-agent*_amd64.deb
 /test/e2e/scripts/setup-instance/id_rsa.pub
 /test/e2e/scripts/setup-instance/ignition.json
 /test/e2e/containers/fake_datadog/venv/
+
+# Cluster Agent's config file
+Dockerfiles/cluster-agent/datadog-cluster.yaml
```

Comment:
Why is this added to the gitignore? it seems the file was already tracked

Reply:
yes, I was planning on moving it in a different spot - I added this here as the `inv cluster-agent.builld` creates a datadog-cluster.yaml at this location. I think `invoke` should not alter the git state. I'll work on that in a different PR.